### ISSUE TITLE
Test the current code in a windows laptop using the sensor

### ISSUE THREAD

**Issue by spsaswat** [spsaswat/APPF_FLIR#2]
Requisite: The thermal camera sensor will be provided to the tester.

Steps to be followed:
1) clone the repository.
2) connect the sensor to the power source and laptop (windows).
3) test the code for a successful camera capture.

**Comment by zzzshuang** [comment]
Hi Saswat @spsaswat, I encountered an issue where the Spinnaker SDK only supports Intel chip Macs, but I'm using a Mac with an M chip, so I can't install the package. Following your previous suggestion(using TeamViewer), I attempted to use ANU Virtual Desktop to install TeamViewer and Spinnaker. Unfortunately, it seems that the ANU virtual desktop doesn't permit the downloading and installation of software. However, I discovered that Parallels Desktop allows me to install Spinnaker, so I am currently attempting to see if I can successfully connect using Parallels Desktop.
